config/sidekiq_scheduler.yml

```diff
@@ -15,3 +15,8 @@ DeleteOldApplications:
 SidekiqStatsJob:
   cron: "* * * * *"
   description: "Update Sidekiq stats for export to statsd gauges"
+
+DeleteOldApplications:
```

Comment:
Also a different key name, please. Seeing this I do agree that having a common DB Record Reaper job would make a lot of sense too, but that can be a future cleanup.